diff --git a/third_party/xla/workspace.bzl b/third_party/xla/workspace.bzl
--- a/third_party/xla/workspace.bzl
+++ b/third_party/xla/workspace.bzl
@@ -37,6 +37,7 @@
             "//third_party/xla:0010-Add-systemlib-for-oneDNN.patch",
             "//third_party/xla:0011-Fix-clang-CUDA-incompatabilities.patch",
             "//third_party/xla:0012-Stub-protoc_minimal-for-native-proto_library.patch",
+            "//third_party/xla:0014-Disable-leave-barriers-flag.patch",
         ],
     )
