From 8f8c5d36414a638ab2e1906975a5b609669a1cdb Mon Sep 17 00:00:00 2001
From: Eric Lundby <Eric.Lundby@gmail.com>
Date: Wed, 28 Jan 2026 14:53:54 -0700
Subject: [PATCH 1/1] 14

---
 .../coordination/coordination_service.cc           | 14 +++++++-------
 1 file changed, 7 insertions(+), 7 deletions(-)

diff --git a/xla/tsl/distributed_runtime/coordination/coordination_service.cc b/xla/tsl/distributed_runtime/coordination/coordination_service.cc
index 8667719a50..ddf8d82056 100644
--- a/xla/tsl/distributed_runtime/coordination/coordination_service.cc
+++ b/xla/tsl/distributed_runtime/coordination/coordination_service.cc
@@ -53,9 +53,9 @@ limitations under the License.
 #include "xla/tsl/protobuf/coordination_service.pb.h"
 #include "xla/tsl/util/device_name_utils.h"

-ABSL_FLAG(bool, leave_barriers_on_recoverable_agent_restart, false,
-          "If true, allow the recoverable agent to leave ongoing barriers on "
-          "restart.");
+// ABSL_FLAG(bool, leave_barriers_on_recoverable_agent_restart, false,
+//           "If true, allow the recoverable agent to leave ongoing barriers on "
+//           "restart.");

 namespace tsl {
 namespace {
@@ -312,7 +312,7 @@ CoordinationService::GetCountOfOutOfSyncTasksPerBarrier() {
 }

 void CoordinationService::CheckBarrierStatusWithRecoverableTasks() {
-  if (!absl::GetFlag(FLAGS_leave_barriers_on_recoverable_agent_restart)) {
+  if (!false) {
     return;
   }

@@ -555,7 +555,7 @@ void CoordinationService::ConnectTask(const CoordinatedTask& task,
   task_state->Connect();
   if (task_state->IsRecoverable()) {
     LeaveOngoingBarriers(task, "recoverable task silently connected again");
-    if (absl::GetFlag(FLAGS_leave_barriers_on_recoverable_agent_restart)) {
+    if (false) {
       unsynced_recoverable_jobs_.insert(task_name);
     }
   }
@@ -619,7 +619,7 @@ void CoordinationService::RegisterTaskAsync(const CoordinatedTask& task,
       // again. When the task passes a barrier with other tasks, it will be
       // removed from the unsynced set.
       if (task_cluster_state->IsRecoverable() &&
-          absl::GetFlag(FLAGS_leave_barriers_on_recoverable_agent_restart)) {
+          false) {
         if (barriers_.contains(kClusterRegisterBarrierId) &&
             barriers_[kClusterRegisterBarrierId].passed) {
           // We want to mark the task unsynced when it connects again. When the
@@ -1723,7 +1723,7 @@ void CoordinationService::LeaveOngoingBarriers(const CoordinatedTask& task,
     for (const auto& barrier_id : task_state->GetOngoingBarriers()) {
       BarrierState* barrier = &barriers_[barrier_id];
       // Unregister task from barrier.
-      if (absl::GetFlag(FLAGS_leave_barriers_on_recoverable_agent_restart)) {
+      if (false) {
         if (barrier->tasks_at_barrier.contains(task)) {
           // Remove task from barrier.
           barrier->recoverable_tasks_restarted_during_barrier.insert(task);
--
2.50.1 (Apple Git-155)
